biz.aQute.resolve/testdata/enroute/.gitignore

```diff
@@ -0,0 +1,3 @@
+/generated/
```

Comment:
This implies the tests should copy the testdata to generated and modify it there. We really, really, really should not have the test cases writing to folders which are not in the output folders. This messes up the gradle incremental building.

Reply:
done